### DISCUSSION TITLE
Is Typst FOSS?

### DISCUSSION THREAD

**Discussion by aberges-grd** [typst/typst#249] [General]
One of the things that made LaTeX so pervasive is that it's FOSS. A single programmer did a incredible piece of software and released it to the world, no price asked.

Same goes for pandoc. Many of us have grown accustomed to pandoc + LaTeX for most of our typesetting, and Typst is literally competing with that.

So my question is: is this a corporate open source product that will be sold, or is it a FOSS project like those 2?

**Comment by alerque** [comment]
https://github.com/typst/typst/blob/main/LICENSE
